internal/provisioner/kops_provisioner.go

```diff
@@ -105,7 +106,8 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster) error
 		return err
 	}
 
-	err = os.Rename(kops.GetOutputDirectory(), outputDir)
+	// err = os.Rename(kops.GetOutputDirectory(), outputDir)
```

Comment:
I would really like to attempt the `os.Rename` and then fall back to `copyFolder` if `err != nil` here.

Looking at all of the edge cases for copying files in go, it would be nice to try to do it cleanly first.

Reply:
@gabrieljackson let me know if the change is good